src/components/Editor/Preview/Popup.css

```diff
@@ -15,12 +15,18 @@
   box-shadow: 1px 2px 3px var(--popup-shadow-color);
 }
 
-.popover .preview-popup .header {
+.popover .preview-popup .header-container {
   width: 100%;
   line-height: 20px;
   border-bottom: 1px solid #cccccc;
   display: flex;
   flex-direction: column;
+  margin-bottom: 5px;
+}
+
+.popover .preview-popup .header-container h3 {
+  margin: 0;
+  margin-bottom: 5px;
```

Comment:
lets soften the header a bit...
smaller font, normal weight...

Reply:
Kay kay how's this:
![react_display_name_soft](https://user-images.githubusercontent.com/16711897/31503610-c3cc0ce0-af35-11e7-86d8-12116ddcd97b.png)

Reply:
Looks better to me. As a small detail for the screens, can you turn off the feature w/ the bottom bar :) it's not ready to ship yet

Reply:
also, this is a nit, but can we go drop the border below App? might be nice

Reply:
> "can you turn off the feature w/ the bottom bar"

Do you mean the bottom bar of the preview?

Reply:
yeah - it's in launchpad, settings "Add watch expression from preview"